src/main/java/core/basesyntax/domain/Fruit.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.domain;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class Fruit {
+    private Operation operation;
```

Comment:
The 'operation' field does not belong to the 'Fruit' domain model. Domain models should be kept simple and focused on representing an entity. If you need to associate operations with the fruit, consider creating a separate class or a service for handling such operations.